### PR TITLE
Fixes typo in code example

### PR DESCRIPTION
**Pull request by JenEEsquivel**
:green_heart: Needs two thumbs up. 

In regards to Issue https://github.com/LMHT/LMHT.github.io/issues/121

### CODE REVIEW

File: elements/link.html [hunk 1]
```diff
@@ -40,7 +40,8 @@
     <code>
       <strong>&lt;html&gt;</strong>
         &lt;head&gt;
-          &lt;title&gt;Document title&lt;/title&gt;
+        &lt;link rel="stylesheet" href="//code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css"&gt;
```
Comment (rylnd) [inline elements/link.html]: Please indent these elements, as they are nested within the `head` element.

File: elements/link.html [hunk 2]
```diff
@@ -40,7 +40,8 @@
...
+        &lt;link type='text/css' href='css/style.css' rel='stylesheet' /&gt;
```
Comment (rylnd) [inline elements/link.html]: This commit adds one link with attribute values in single quotes, and another with them in double quotes. Strive for consistency in style, especially within a particular file.

**Comment by rylnd** [comment]
Good catch on the content for this page! I have a few minor style nitpicks, but once those are addressed you have my :+1:

**Comment by ericanaughton** [comment]
@JenEEsquivel how's it coming along?

**Comment by ericanaughton** [comment]
The wrong tag is highlighted. < link > should be highlighted, not < html >.
image

I suppose it doesn't relate to this issue though.

**Comment by bigtiger** [comment]
@JenEEsquivel @ericanaughton If the wrong thing was highlighted before then :+1: and add an issue for the next sprint.

**Comment by ericanaughton** [comment]
The title of this PR relates to Issue https://github.com/LMHT/LMHT.github.io/issues/97, so you may want to rename this to something like "Fixes typo in code example"

**Comment by ericanaughton** [comment]
:+1:

**Comment by rylnd** [comment]
:+1:

**Pull request merged**